res/database/Default/user/scr0llbaer_constr_d2v_tv_series.xml

```diff
@@ -0,0 +1,6397 @@
+				<en>Dynamite - The Magazine Live: Wedding of [3|First] und [2|Full]</en>
+			</title>
+			<description>
+				<de>Der 53-jährige Fürst [2|Full] und die 20-jährige Punk-Pinzessin [3|First] heiraten in Regensburg, ja kann das gut gehen? Wie schön ist's im Schloss? Ist es wirklich Liebe? Braucht der Fürst Hilfsmittel zum Schnackseln? Fragen über Fragen.</de>
+				<en>The 53-year-old Prince [2|Full] and the 20-year-old Punk Princess [3|First] get married in Regensburg, can that go well? How beautiful is it in the castle? Is it really love? Does the prince need aids for bonking? Questions upon questions.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">cebd1a68-cf97-4b93-b3f9-ac34bda4bdcc</member>
+				<member index="1" function="136">88ba23dc-d4f1-4c77-9ddf-8dbf4a4ce3b2</member>
+				<member index="2" function="64">9d987fbb-99c3-47b8-82c3-227461b6762c</member>
+				<member index="3" function="64">64828f7e-a96b-4bb3-85c6-477ebe408d05</member>
+			</staff>
+			<data country="D" distribution="2" maingenre="301" subgenre="6,15" flags="17" blocks="6" price_mod="0.5" />
+			<releaseTime year="1980" day="5" hour="10" />
+			<ratings critics="15" speed="30" outcome="60" />
+			<modifiers>
+				<modifier name="topicality::notLive" value="0.25" />
+			</modifiers>
+		</programme>
+		<programme id="583955ef-3acf-42bf-991e-b1c169583780" product="4" licence_type="1" fictional="1" creator="9551">
+			<title>
+				<de>Dynamit - Das Magazin Live: Hochzeit von [2|Nick] und [3|Nick]</de>
+				<en>Dynamite - The Magazine Live: Wedding of [2|Nick] und [3|Nick]</en>
+			</title>
+			<description>
+				<de>In der Westminster Abbey ist nicht genug Platz. Der 32-jährige [2|Nick] und die 20-jährige [3|Nick] heiraten daher in der Londoner St Paul's Cathedral. Ja kann das gut gehen? Ex-Model [1|Full] berichtet live.</de>
+				<en>There is not enough space in Westminster Abbey. So 32-year-old [2|Nick] and 20-year-old [3|Nick] get married in London's St Paul's Cathedral. Can that go well? Ex-model [1|Full] reports live.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">cebd1a68-cf97-4b93-b3f9-ac34bda4bdcc</member>
+				<member index="1" function="136">88ba23dc-d4f1-4c77-9ddf-8dbf4a4ce3b2</member>
+				<member index="2" function="64">28fe86a0-74ca-43a7-bc49-d3bd4df7c4d3</member>
+				<member index="3" function="64">4623935a-750a-4e5a-9d1b-6875062ff2a8</member>
+			</staff>
+			<data country="D" distribution="2" maingenre="301" subgenre="6,15" flags="17" blocks="6" price_mod="0.5" />
+			<releaseTime year="1981" day="7" hour="10" />
+			<ratings critics="20" speed="40" outcome="95" />
+			<modifiers>
+				<modifier name="topicality::notLive" value="0.25" />
+			</modifiers>
+		</programme>
+		<programme id="7df480c6-93c3-4616-8e76-d3d697f9445f" product="7" licence_type="3" fictional="1" creator="9551" comment="asked ChatGPT for a little help creating this, came back with good results">
```

Comment:
There have been ChatGPT examples floating around already, so I thought it would be fine.

If there's an issue it might be that ChatGPT is not "free", which bothers me as well, but I didn't come around to install a FLOSS chat AI thing locally yet (I only have Midjourney running so far).

Reply:
The main point is. Yes, please annotate the entries (and encourage others to do so), so there is a chance of finding them in case of policy changes.